apps/web/src/pages/ProfileAndApplicationsPage/ProfileAndApplicationsPage.tsx

```diff
@@ -14,6 +14,12 @@ import ProfileAndApplicationsHeading from "./components/ProfileAndApplicationsHe
 import TrackApplications from "./components/TrackApplications/TrackApplications";
 import { PartialUser } from "./types";
 
+const description = defineMessage({
```

Comment:
Can't you move this into line 37 and get rid of setting this variable as well?

Reply:
Oh we're moving to defining messages like this outside the component (just above usually) so they aren't getting redeclared if rerendered.

Reply:
Oh yeah, I was the one who said that :woman_facepalming: Though, I probably said it in a confusing way. I meant we should do that if we need to  reuse it in different spots :sweat_smile: if it is just once i the render function it should be fine

Reply:
😆 Okay, changed in [08cfed5](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9839/commits/08cfed511e7ccd223927587761a801c2ee1c3d67)